[2021-12-06 13:47:57,580] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 2021-12-06T13:47:23.616328+00:00 [queued]>
[2021-12-06 13:47:57,821] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 2021-12-06T13:47:23.616328+00:00 [queued]>
[2021-12-06 13:47:57,822] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-06 13:47:57,823] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-06 13:47:57,824] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-06 13:47:57,939] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_3> on 2021-12-06T13:47:23.616328+00:00
[2021-12-06 13:47:57,989] {standard_task_runner.py:52} INFO - Started process 55881 to run task
[2021-12-06 13:47:58,086] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_rule', 'task_3', '2021-12-06T13:47:23.616328+00:00', '--job-id', '754', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/trigger_rule.py', '--cfg-path', '/tmp/tmp94pwe2yu', '--error-file', '/tmp/tmp9gd858bo']
[2021-12-06 13:47:58,452] {standard_task_runner.py:77} INFO - Job 754: Subtask task_3
[2021-12-06 13:47:59,403] {logging_mixin.py:104} INFO - Running <TaskInstance: trigger_rule.task_3 2021-12-06T13:47:23.616328+00:00 [running]> on host airflowvm
[2021-12-06 13:48:00,116] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=trigger_rule
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2021-12-06T13:47:23.616328+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-06T13:47:23.616328+00:00
[2021-12-06 13:48:00,119] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-06 13:48:00,143] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'exit 0']
[2021-12-06 13:48:00,165] {subprocess.py:75} INFO - Output:
[2021-12-06 13:48:00,258] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-06 13:48:00,324] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=trigger_rule, task_id=task_3, execution_date=20211206T134723, start_date=20211206T134757, end_date=20211206T134800
[2021-12-06 13:48:00,682] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-12-06 13:48:00,690] {local_task_job.py:151} INFO - Task exited with return code 0
